src/components/Greetings/index.tsx

```diff
@@ -1,12 +1,13 @@
 import React from 'react'
 
 import { Container, Image, Text } from './styles'
+import logo from '../../assets/logo.svg'
```

Comment:
I thought about importing by the `<img/>` tag because of the _**alt**_ attribute, but thinking here, I think it's the responsibility of the creator of svg to add a title for their svg to have accessibility